### DISCUSSION TITLE
Adding exemptions for SC 2.4.1, 2.4.5, 3.2.3, and 3.2.4 in WCAG2ICT

### DISCUSSION THREAD

**Discussion by samogami** [w3c/wcag2ict#50] [General]
Both [US Section 508](https://www.access-board.gov/ict/#b-application-of-wcag-2-0-to-non-web-ict) and [EN 301 549](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf) have exemptions to applying four WCAG 2.0 SC:
* [2.4.1 Bypass Blocks](https://www.w3.org/TR/wcag2ict/#navigation-mechanisms-skip)
* [2.4.5 Multiple Ways](https://www.w3.org/TR/wcag2ict/#navigation-mechanisms-mult-loc)
* [3.2.3 Consistent Navigation](https://www.w3.org/TR/wcag2ict/#consistent-behavior-consistent-locations)
* [3.2.4 Consistent Identification](https://www.w3.org/TR/wcag2ict/#consistent-behavior-consistent-functionality) 

These exemptions should also be added to the updated WCAG2ICT.

**Comment by maryjom** [comment]
@sam I have added issue #145  for this.
